### DISCUSSION TITLE
Welcome to HotelManagement Discussions!

### DISCUSSION THREAD

**Discussion by adnanhasan251** [adnanhasan251/HotelManagement#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JDeepD** [comment]
Haa. Oie Code dalega na pdf mai?

**Reply by JDeepD** [reply]
@adnanhasan251

**Reply by adnanhasan251** [reply]
Sorry meko notifications nahi milta iska issiliye meko aaj dikha yeh

**Comment by Mastermind-sap** [comment]
op output🚀!!!now my project looks so dull😭

**Reply by JDeepD** [reply]
??

**Reply by Mastermind-sap** [reply]
kuch nahi  meant adnan ka project ka output bohot acha hai...in the sense full coloured stuff in cli...tera toh gui hai toh tera toh bohot acha hai lekin cui mei adnan ka presentation mei marks de dega...mera outpur is very bad😫🤢

**Reply by JDeepD** [reply]
Tujhe kya lgta hai project dekhega bhi?😂😂. Chemistry ki tarah lega project. Ye sab toh bas formality hai marks ka. Real mai sabhi ko 28+ hi deta hai school.

**Reply by Mastermind-sap** [reply]
oh...lekin mereko bohot dukh hoga agr nhi dekha toh😅

**Reply by JDeepD** [reply]
Tu kya laptop leke jayega?

**Reply by JDeepD** [reply]
Code ka output dikhane ke liye?

**Reply by Mastermind-sap** [reply]
nhi...output print kr dega bsss...laptop leke kaun jayega aur wese bhi nahi dekhega run karwa kr shayad...tu kya laptop leke jayega?

**Reply by Mastermind-sap** [reply]
@JDeepD baap re tu deep bhomik ka profile nikala kya stackexchange mei....mtlab tu bhi hai😅...wese woh comment delete kyu kiya

**Reply by JDeepD** [reply]
@Mastermind-sap https://askubuntu.com/users/915987/rupesh

Rupesh Jha???????????

**Reply by Mastermind-sap** [reply]
bhai bhai😂😂😂 @JDeepD  tu kaha se nikal rha hai yeh sab....lekin rupesh sir ka naam shayad rupesh kumar jha hai not krishna😂

**Reply by JDeepD** [reply]
Krishna bhi ho skta hai

**Reply by JDeepD** [reply]
Mereko tera account bhi mil gaya

**Reply by Mastermind-sap** [reply]
mei hu hi nai😂 stackexchange mei

**Reply by Mastermind-sap** [reply]
kya tu script use krke nikaal tha hai yaah search kr rha hai manually?

**Reply by JDeepD** [reply]
Nhi mil gaya mujhe

**Reply by JDeepD** [reply]
@Mastermind-sap  Rocky https://physics.stackexchange.com/users/241080/rocky-saikia

**Reply by JDeepD** [reply]
Mujhe bol ye tu nhi hai https://stackoverflow.com/questions/65904594/how-to-make-akinator-command-in-discord-py/65957068#65957068

**Reply by JDeepD** [reply]
> Mujhe bol ye tu nhi hai https://stackoverflow.com/questions/65904594/how-to-make-akinator-command-in-discord-py/65957068#65957068

@Mastermind-sap

**Reply by JDeepD** [reply]
@Mastermind-sap 
Deleted Acc : https://stackoverflow.com/users/14586140/user14586140
Current  acc : https://stackoverflow.com/users/15106982/mastermindsap

Hai na?

**Reply by JDeepD** [reply]
@Mastermind-sap Aur ek baat. Stack Overflow bohot chalak hai. Wo ip tracing se same person different accounts ka pta lagake merge kr dete hai. Mera 150rep cut ho gaya tha uski waje se ek baar . Aur phirse krne se site ban laga deta hai beh😂😂😂

**Reply by JDeepD** [reply]
@Mastermind-sap aajkal linkedin pe nhi hai kya tu?

**Reply by Mastermind-sap** [reply]
> @Mastermind-sap Rocky https://physics.stackexchange.com/users/241080/rocky-saikia

nice


> Mujhe bol ye tu nhi hai https://stackoverflow.com/questions/65904594/how-to-make-akinator-command-in-discord-py/65957068#65957068

oh yeh mera sif ek activity hai .....only time i used stackoverflow sif iska reply krne ke liye bnaya tha.

> @Mastermind-sap
> Deleted Acc : https://stackoverflow.com/users/14586140/user14586140
> Current acc : https://stackoverflow.com/users/15106982/mastermindsap
> 
> Hai na?

nhi upr wala mera nhi hai

> @Mastermind-sap Aur ek baat. Stack Overflow bohot chalak hai. Wo ip tracing se same person different accounts ka pta lagake merge kr dete hai. Mera 150rep cut ho gaya tha uski waje se ek baar . Aur phirse krne se site ban laga deta hai beh

pata nhi mei kabhi use nhi kiya.....

> @Mastermind-sap aajkal linkedin pe nhi hai kya tu?

mei kabhi likedin mei nhi tha...profile shayad class 7 yah 8 mei create kiya hoga...

@JDeepD tera profiles ka link de(of stackoverflow and linkedin)

**Reply by JDeepD** [reply]
https://stackexchange.com/users/16755257/jaydeep-das?tab=accounts
@Mastermind-sap

**Reply by Mastermind-sap** [reply]
nice...tera questions zero kyu dikha rha hai jbki mereko ek bootable device wala question mila jo tu pucha?

**Reply by JDeepD** [reply]
0 net votes

**Reply by Mastermind-sap** [reply]
kismei?

**Reply by adnanhasan251** [reply]
Me to this discussion channel:
Asli id se aa.. WhatsApp

**Reply by Mastermind-sap** [reply]
@JDeepD leaves replying to the discussion like he left whatsapp and telegram😂

**Reply by subnut** [reply]
_Knock knock_

**Reply by Mastermind-sap** [reply]
_Whose there?_

**Reply by JDeepD** [reply]
Bsdi wale chacha

**Reply by JDeepD** [reply]
> nice...tera questions zero kyu dikha rha hai jbki mereko ek bootable device wala question mila jo tu pucha?

Kaunsa mei

**Reply by Mastermind-sap** [reply]
waah. kal ka para mereko yaad nhi rehta aur tu mereko do mahine pehle ke baat ka rely krke _kaunsa_ puch rha hai :laughing:

**Reply by subnut** [reply]
> Bsdi wale chacha

@JDeepD  
_Bsdi_ ?? _Bsdi_ meaning?

**Reply by JDeepD** [reply]
> > Bsdi wale chacha
> 
> @JDeepD
> _Bsdi_ ?? _Bsdi_ meaning?

**Berkeley Software Distribution** for **Individuals**

![bhosdiwale-chacha-meme-template-mirzapur](https://user-images.githubusercontent.com/64089730/121769380-f3b65f80-cb80-11eb-90e3-54354f269dcf.jpg)
